symbolic-debuginfo/src/elf.rs

```diff
@@ -669,6 +674,15 @@ impl<'data> ElfObject<'data> {
             }
         }
 
+        const PT_SCE_DYNLIBDATA: u32 = 0x61000000;
+
+        for ph in &self.elf.program_headers {
+            if ph.p_type == PT_SCE_DYNLIBDATA && ph.p_filesz >= 20 {
```

Comment:
What's the significance of 20 here?

Reply:
From what information I can find on the public web, this section starts with a 24 byte `signature`, but normal build ids are only 20 bytes. In the files I am looking at, we actually have 16 usable bytes, with 4 null-bytes afterwards.